doc/local-dev.md

```diff
@@ -0,0 +1,101 @@
+#### Goal
+The following steps are going to set up a dev environment with minikube. This is meant to allow you to test your changes to draughtsman before they are committed or merged.
+
+*The usage guide is more detailed about actually running draughtsman. This guide is only meant to set up a local environment to develop on draughtsman!*
+
+#### Preparation
+
+The following steps are meant to prepare your minikube and the environment `draughtsman` is going to need. You should have `minikube` and docker installed on your machine beforehand.  
+Start up your minikube, here is an example configuration (note that `k8s v1.7.3` caused some errors in our setups):  
+`minikube start --cpus 4 --memory 4096 --kubernetes-version v1.7.0`  
+Your kubectl should be configured to use your minikube now and kubernetes should be available at `https://192.168.99.100:8443`.  
+Next install helm (see installation guides [here](https://github.com/kubernetes/helm/blob/master/docs/install.md)) and run `helm init` against your minikube. Check if tiller is successfully deployed with:  
+ `kubectl describe deployment tiller-deploy -n=kube-system`.
+
+A `secret.yaml` has to be prepared next, which contains all information for draughtsman to connect to your minikube and other entities:
+```
+service:
+  kubernetes:
+    incluster: false
+    address: https://192.168.99.100:8443
+    tls:
+      cafile: /minikube/apiserver.crt
+      crtfile: /minikube/apiserver.crt
+      keyfile: /minikube/apiserver.key
+  deployer:
+    environment: minikube
+    eventer:
+      github:
+        oauthtoken: your_token
+        organisation: your_org
+        projectlist: your_project, your_other_project
+    installer:
+      helm:
+        organisation: your_quay_org
+        password: your_quay_password
+        username: your_quay_username
+    notifier:
+      slack:
+        channel: your_channel
+  slack:
+    token: your_slack_token
+```
+Note that *all* of these values are mandatory! `Draughtsman` will not work if one or several of these values are missing!
```

Comment:
Alternatively:

    `Draughtsman` will not work if any of these values are missing